ApplicationLibCode/ProjectDataModel/Summary/RimDeltaSummaryCase.cpp

```diff
@@ -188,12 +239,12 @@ void RimDerivedSummaryCase::calculate( const RifEclipseSummaryAddress& address )
 //--------------------------------------------------------------------------------------------------
 ///
 //--------------------------------------------------------------------------------------------------
-std::pair<std::vector<time_t>, std::vector<double>> RimDerivedSummaryCase::calculateDerivedValues( RifSummaryReaderInterface* reader1,
-                                                                                                   int fixedTimeStepCase1,
-                                                                                                   RifSummaryReaderInterface* reader2,
-                                                                                                   int fixedTimeStepCase2,
-                                                                                                   DerivedSummaryOperator m_operator,
-                                                                                                   const RifEclipseSummaryAddress& address )
+std::pair<std::vector<time_t>, std::vector<double>> RimDeltaSummaryCase::calculateDerivedValues( RifSummaryReaderInterface* reader1,
+                                                                                                 int fixedTimeStepCase1,
+                                                                                                 RifSummaryReaderInterface* reader2,
+                                                                                                 int                    fixedTimeStepCase2,
+                                                                                                 DerivedSummaryOperator m_operator,
```

Comment:
Strange indentation for fixedTimeStepCase2, but if clang-format is happy I guess it is correct.

Reply:
Clang is happy